service_contracts.rst

```diff
@@ -150,9 +154,12 @@ expiration_block).
 .. [#sig] The signature is calculated by
           ::
 
-               ecdsa_recoverable(privkey,
-                                 sha3_keccak("\x19Ethereum Signed Message:\n104"
-                                             || sender || receiver || amount || expiration_block ))
+               ecdsa_recoverable(
+                   privkey,
+                   sha3_keccak("\x19Ethereum Signed Message:\n104"
```

Comment:
The size needs to be adapted here.

Reply:
Good catch! I'll fix it.

Reply:
Did it happen in the implementation?  Looking...  :eyes:

Reply:
Yes, it did happen there.